lib/components/form/LocationInput.tsx

```diff
@@ -1,262 +1,215 @@
-import 'mapbox-gl/dist/mapbox-gl.css';
-import React, { useRef, useState, useEffect, FC } from 'react';
-import mapboxgl from 'mapbox-gl';
 import MapboxGeocoder from '@mapbox/mapbox-gl-geocoder';
+import mapboxgl from 'mapbox-gl';
+import 'mapbox-gl/dist/mapbox-gl.css';
+import { FC, useEffect, useRef, useState } from 'react';
 import { LatitudeLongitude } from '~/lib/types';
+import { reverseGeocodeWithoutFocus } from '~/lib/utils';
+
+mapboxgl.accessToken = process.env.NEXT_PUBLIC_MAPBOX_PUBLIC!;
 
 interface LocationInputProps {
-  originalLng: number;
-  originalLat: number;
-  shouldZoomIntoOriginal: boolean;
-  setIsSubmissionEnabled: (isSubmissionEnabled: boolean) => void;
```

Comment:
removing this allows anything to be inputted in the map—next button is enabled even when outside zoom threshold and also when completely outside of the UK